app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -33,22 +33,15 @@
 
     public static CoverFragment newInstance(Playable item) {
         CoverFragment f = new CoverFragment();
-        if (item != null) {
-            Bundle args = new Bundle();
-            args.putParcelable(ARG_PLAYABLE, item);
-            f.setArguments(args);
-        }
+        f.media = item;
```

Comment:
Why did you remove the bundle? Afaik using bundles is a best practice: http://stackoverflow.com/a/9245510/5369600 and https://developer.android.com/reference/android/app/Fragment.html